index.js

```diff
@@ -50,16 +49,8 @@ function AtImport(options) {
   return function(styles, result) {
     const opts = assign({}, options || {})
     // auto add from option if possible
-    if (
-      !opts.from &&
-      styles &&
-      styles.nodes &&
-      styles.nodes[0] &&
-      styles.nodes[0].source &&
-      styles.nodes[0].source.input &&
-      styles.nodes[0].source.input.file
-    ) {
-      opts.from = styles.nodes[0].source.input.file
+    if (!opts.from && styles.source.input.file) {
+      opts.from = styles.source.input.file
```

Comment:
Not of the scope of the PR. Please revert this. Issues already happened and will happen again when people play with AST and introduce mistakes.